### DISCUSSION TITLE
MarkdownTextBlock not available from Labs-Windows feed?

### DISCUSSION THREAD

**Discussion by euju-ms** [CommunityToolkit/Labs-Windows#591] [Q&A] [answered]
I added the Labs-Windows feed to my Package Sources list from VisualStudio, but I can't find the MarkdownTextBlock.. anyone know what's going on? (I used the Labs-Windows feed: `https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-Labs/nuget/v3/index.json`)

Searching for `Markdown` results in nothing.

![image](https://github.com/user-attachments/assets/8e11bbe2-d1f4-404c-924b-c212bdf9a891)

**Comment by beeradmoore** [comment, marked as answer]
I think you need to hit the "Include pre-release" checkbox.

**Reply by euju-ms** [reply]
Ah... thank you so much! I didn't realize this was in a pre-release... but I should've tried it anyways :)
